debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -238,6 +238,17 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
     public static final long DEFAULT_POLL_INTERVAL_MILLIS = 500;
     public static final String DATABASE_CONFIG_PREFIX = "database.";
     private static final String CONVERTER_TYPE_SUFFIX = ".type";
+    public static final int DEFAULT_RETRIABLE_RESTART_WAIT = 10;
+
+    public static final Field RETRIABLE_RESTART_WAIT = Field.create("retriable.restart.connector.wait.sec")
```

Comment:
Could you change sec to ms, please? It's the general pattern used for all duration-typed options. This also should be added to `CONFIG_DEFINITION`.

Reply:
Yep.